src/main/java/com/javadiscord/javabot/service/AutoMod.java

```diff
@@ -24,6 +31,22 @@
  */
 public class AutoMod extends ListenerAdapter {
 
+    private static List<String> spamUrls;
+
+    public AutoMod() {
+        try {
+            Scanner scanner = new Scanner(new File(String.valueOf(getClass().getResourceAsStream("spamsLinks.txt"))));
```

Comment:
And on your last PR Andrew said you shouldn't use a Scanner for this; As he didn't want me to change my approach you _could_ use [Files#getAllLines](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#readAllLines-java.nio.file.Path-)